kop/java/README.md

```diff
@@ -64,3 +66,128 @@ See [KoP Security](https://github.com/streamnative/kop/blob/master/docs/security
    ```
    Receive record: hello from persistent://public/default/my-topic-0@0
    ```
+
+## Example: OAuth2 authentication 
+
+See [KoP Security](https://github.com/streamnative/kop/blob/master/docs/security.md#oauthbearer) for how to configure KoP with oauth authentication. This example takes a topic named `my-topic` under `public/default` namespace as reference.
+1. Start the hydra OAuth2 Server
```

Comment:
```suggestion
1. Start the Hydra OAuth2 server.
```
Another 2 comment on this step:
1. Either you use "Start and initialize the Hydar OAuth2 server." and combine the following two commands in to one code block. Or, you use sub-steps for the start and initialize operations.
2. Pay attention to the indent of code/ context under each step. Please check through the whole doc and update them one by one.